packages/shared/lib/i18n.ts

```diff
@@ -147,8 +147,16 @@ const setLanguage = (item) => {
     setupI18n({ withLocale: locale })
 }
 
+const getDecimalSeparator = () => {
+    const numberWithDecimalSeparator = 1.1;
+    return Intl.NumberFormat(get(appSettings).language)
```

Comment:
I feel like we should use the browser locale here instead so that it matches the system settings. For example, in Panama (`es-PA`) the decimal separator is `.` but in Argentina (`es-AR`) the decimal separator is `,`. Using the language code from our app (`Spanish (Latin America)` is `es-419`) would make both use `.`.

```suggestion
    return Intl.NumberFormat(getLocaleFromNavigator())
```

Reply:
I wasn't sure which was the preferred behavior, if someone had specifically picked a language from settings, if that would take priority - but since we dont support all languages you might have no choice. Will go with your suggestion for now.